### DISCUSSION TITLE
Force ASCII table to show tbl-cap(tion) with rtables

### DISCUSSION THREAD

**Discussion by dmenne** [quarto-dev/quarto-cli#3255] [Q&A] [answered]
Package `rtables` creates ASCII tables; when I convert these to `flextable `(with `tt_to_flextable`), the table caption shows.

How do I force Quarto to show a table caption with an ASCII table as produced by `rtables`?

`````
# Force Table 


```{r}
#| echo: false
#| message: false
#| tbl-cap: Age table (this text is not shown)
library(rtables)
library(flextable)
lyt = basic_table() %>%
  analyze("AGE") %>% 
  build_table(DM)
lyt
```

How do I say: consider this a table and add the caption?

```{r}
#| echo: false
#| tbl-cap: Age table 
 lyt %>% 
  tt_to_flextable()
```

**Comment by dmenne** [comment]
![2022-11-10 16_28_50-mybook - 2  Summary](https://user-images.githubusercontent.com/506275/201137048-55a2eb56-3383-47f9-ba9d-787b0bc266b1.png)

**Comment by cscheid** [comment, marked as answer]
> How do I say: consider this a table and add the caption?

Unfortunately that's not possible in the current version of quarto. We'll be working on this in 1.3, though, so stay tuned!

**Reply by dmenne** [reply]
Any progress? Checking builds every day .-)

**Reply by cscheid** [reply]
When we make progress, we'll both close these issues and put the information on the prerelease page. We're still working on it.

(Please don't take this the wrong way either, and we super appreciate the enthusiasm!) But every question like this is time we don't get to spend actually working on the issues :)
